examples/shader/shader_defs.rs

```diff
@@ -1,81 +1,52 @@
-//! A shader that uses "shaders defs" (a bevy tool to selectively toggle parts of a shader)
```

Comment:
I'm not sure why this was changed. This examples does show how to use shader defs so adding it to the top level description seems relevant.

Reply:
Oops this was a copy/paste issue. The old text was definitely better!